File: task-03/Go/subtask-02.go
package main

import (
	"io/ioutil"
	"log"
)

func main() {

	input, err := ioutil.ReadFile("input.txt")
	if err != nil {
		log.Fatal(err)
	}

	err = ioutil.WriteFile("output.txt", input, 0644)
	if err != nil {
		log.Fatal(err)
	}
}

File: task-03/C/subtask-02.c
#include <stdio.h>

int main() {
    FILE *inputFile, *outputFile;
    char buffer[1024];

    inputFile = fopen("input.txt", "r");
    if (inputFile == NULL) {
        perror("Error opening input file");
        return 1;
    }

    outputFile = fopen("output.txt", "w");
    if (outputFile == NULL) {
        perror("Error opening output file");
        fclose(inputFile);
        return 1;
    }

    while (fgets(buffer, sizeof(buffer), inputFile) != NULL) {
        fputs(buffer, outputFile);
    }

    fclose(inputFile);
    fclose(outputFile);

    return 0;
}
